### PR TITLE
xemu: 0.7.85 -> 0.7.87

### PR DESCRIPTION
**Pull request by lasers**
###### Description of changes
Bump version.

If forgotten, `echo "d8fa50e524c22f85ecb2e43108fd6a5501744351" > XEMU_COMMIT` will be wrong... (see https://github.com/NixOS/nixpkgs/pull/220275), I move this up.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/emulators/xemu/default.nix [hunk 1]
```diff
@@ -89,20 +89,19 @@ stdenv.mkDerivation (self: {
...
-    # When the data below can't be obtained through git, the build process tries
-    # to run `XEMU_COMMIT=$(cat XEMU_COMMIT)` (and similar)
-    branch = "master";
-    commit = "d8fa50e524c22f85ecb2e43108fd6a5501744351";
-    inherit (self) version;
-  in ''
+   preConfigure = let
+        # When the data below can't be obtained through git, the build process tries
+        # to run `XEMU_COMMIT=$(cat XEMU_COMMIT)` (and similar)
+        rev = "4a99fd0f1855fc52e15885812abd3ee4bb2b3847";
```
Comment (lasers) [inline pkgs/applications/emulators/xemu/default.nix:95]: How can we get `rev` hash from `src = fetchFromGitHub { rev = "v${version}" }` for this?

Comment (AndersonTorres) [reply pkgs/applications/emulators/xemu/default.nix]: Usually nix-prefetch scripts can do this. I need to seek an example code, tho.

File: pkgs/applications/emulators/xemu/default.nix [hunk 2]
```diff
@@ -27,13 +27,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "xemu";
-  version = "0.7.85";
+  version = "0.7.86";
 
   src = fetchFromGitHub {
     owner = "xemu-project";
-    repo = "xemu";
+    repo = finalAttrs.pname;
```
Comment (AndersonTorres) [inline pkgs/applications/emulators/xemu/default.nix:34]: Leave it as before. There is no reason to parameterize `pname`.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/emulators/xemu/default.nix (hunk 1 above)
Comment (lasers) [reply pkgs/applications/emulators/xemu/default.nix]: (Alternative). Don't bother putting in `master` and `rev`. A version alone is more than enough to fix the title.

Comment (lasers) [reply pkgs/applications/emulators/xemu/default.nix]: No answer. I put in alternative (omitting `master` and `rev`). New version bump too. Lmk.

**Pull request merged**